src/impl/transact_log_handler.cpp

```diff
@@ -279,7 +310,7 @@ class TransactLogValidationMixin {
     bool insert_empty_rows(size_t, size_t, size_t, bool) { return true; }
     bool erase_rows(size_t, size_t, size_t, bool) { return true; }
     bool swap_rows(size_t, size_t) { return true; }
-    bool clear_table() noexcept { return true; }
+    bool clear_table(size_t) noexcept { return true; }
```

Comment:
It looks like your core PR that changes this method signature is based on `develop`, meaning it'll only be available in core v3.x. Can we maintain compatibility with core v2.x without too much extra hassle?